src/components/LoadDatasetButton.tsx

```diff
@@ -77,22 +233,48 @@ export default function LoadDatasetButton(props: LoadDatasetButtonProps): ReactE
       <Modal
         title={"Load a single dataset or collection"}
         open={isLoadModalOpen}
-        okText={"Load"}
-        onOk={handleLoadClicked}
-        okButtonProps={{ loading: isLoading }}
         onCancel={handleCancel}
-        cancelButtonProps={{ hidden: true }}
         getContainer={modalContextRef.current || undefined}
         afterOpenChange={(open) => open && inputRef.current?.focus({ cursor: "all" })}
+        footer={<Button onClick={handleCancel}>Cancel</Button>}
       >
-        <div style={{ display: "flex", flexDirection: "column", gap: "20px" }}>
-          <p>Load a collection of datasets or a single dataset by providing its URL.</p>
-          <Input
-            placeholder="https://example.com/collection.json"
-            value={urlInput}
-            ref={inputRef}
-            onChange={(event) => setUrlInput(event.target.value)}
-          />
+        <div style={{ display: "flex", flexDirection: "column", gap: "10px" }}>
+          <p style={{ marginBottom: "10px" }}>
+            Load a collection of datasets or a single dataset by providing its URL.
+          </p>
+
+          <div ref={dropdownContextRef} style={{ position: "relative" }}>
+            <div style={{ display: "flex", flexDirection: "row", gap: "6px" }}>
+              <Space.Compact style={{ width: "100%" }}>
+                <Dropdown
+                  trigger={["click"]}
+                  menu={datasetsDropdownProps}
+                  placement="bottomLeft"
+                  open={showRecentDropdown}
+                  getPopupContainer={dropdownContextRef.current ? () => dropdownContextRef.current! : undefined}
+                  dropdownRender={renderDropdown}
+                >
+                  <Input
+                    placeholder="https://example.com/collection.json"
+                    value={urlInput}
+                    ref={inputRef}
+                    onChange={(event) => setUrlInput(event.target.value)}
+                    allowClear
+                    disabled={isLoading}
+                  />
+                </Dropdown>
+                <Button type="primary" onClick={handleLoadClicked} loading={isLoading}>
+                  Load
+                </Button>
+              </Space.Compact>
+            </div>
+            <p>
+              <i>
+                <span style={{ color: theme.color.text.hint }}>Click for recent datasets</span>
```

Comment:
Nit: I was clicking on this text expecting a reaction before I realized I should click on the input field, slightly confusing. Could replace example URL text, or a tooltip? Not a big deal.

Reply:
Ended up moving the text on top, and explicitly stating "Click below". Thank you!

Reply:
![image](https://github.com/allen-cell-animated/nucmorph-colorizer/assets/30200665/1e8d70ae-47dd-47da-9510-ce1a3731fcac)